ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -50,5 +121,202 @@
         fr: 'trouvez une zone safe',
       },
     },
+    {
+      id: 'Temple Spirit Wave',
+      regex: / 14:1FE7:Ivon Coeurlfist starts using Spirit Wave/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      infoText: {
+        en: 'AoE',
+      },
+    },
+    {
+      id: 'Temple Touch Of Slaughter',
+      regex: / 14:1FE6:Ivon Coeurlfist starts using Touch Of Slaughter on (\y{Name})/,
+      alertText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Slaughter on YOU',
+          };
```

Comment:
If I remember correctly, there's nothing a player can do to avoid this mechanic. If that's the case, alerting the player seems like noise.

Reply:
There are ground effects at the time this goes out, and it's quickly followed by Hurricane Kick, the 270-degree cleave, so the target needs to be careful until the healer can get to them. Is it perhaps better to pre-alert the target for Hurricane Kick instead?

(I'm thinking of this as being sort of like delayed Hell Winds in E2s, where the target can't do anything about it directly, but *does* have to take different steps to resolve it.)

Reply:
I think Hell Winds are a great example actually; so, I agree with you that the healer probably should have something then that says to heal the slaughter target. However, with Hell Winds, the player has to explicitly change how they act in E2S, either by refusing to stack or not standing in the bleed pillar. Here I don't think the player does anything different than "avoid shit you'd usually avoid." so does that make the `on YOU` trigger valid still?

Reply:
That is a fair point. Nothing does change for the target, just for the healer.